rails/app/controllers/api/v1/jwt_controller.rb

```diff
@@ -35,6 +35,18 @@ def portal
       }
     end
 
+    if (user.has_role? 'admin')
+      claims[:admin] = 1
+    else
+      claims[:admin] = -1
+    end
+    claims[:project_admins] = []
+    user.project_users.each do |p|
+      if(p.is_admin)
+        claims[:project_admins].push(p.project_id)
+      end
+    end
+
```

Comment:
@scytacki this is probably the most dangerous change to the portal in this PR.

Reply:
Good point. I think it is safe enough to try out.